cpu/stm32_common/periph/flashpage.c

```diff
@@ -140,6 +136,11 @@ void flashpage_write_raw(void *target_addr, const void *data, size_t len)
     assert(((unsigned)target_addr + len) <
            (CPU_FLASH_BASE + (FLASHPAGE_SIZE * FLASHPAGE_NUMOF)) + 1);
 
+#ifndef CPU_FAM_STM32F1
+    DEBUG("[flashpage_raw] unlocking the flash module\n");
+    _unlock();
```

Comment:
Is it really true that for all the following boards, it is not necessary to `_unlock` for `flashpage_write_raw` ?

```
cpu/sam0_common/Makefile.features:FEATURES_PROVIDED += periph_flashpage_raw
cpu/stm32f0/Makefile.features:  FEATURES_PROVIDED += periph_flashpage_raw
cpu/stm32l0/Makefile.features:FEATURES_PROVIDED += periph_flashpage_raw
cpu/stm32l1/Makefile.features:FEATURES_PROVIDED += periph_flashpage_raw
```

If yes is it necessary to do `_unlock` for `_erase_page` for them ?

Reply:
Sorry misread the `ifndef`.